GramAddict/core/interaction.py

```diff
@@ -78,6 +74,11 @@ def interact_with_user(
             logger.info("Skip user.", extra={"color": f"{Fore.GREEN}"})
             return False, False
 
+    if scraping_file is not None:
+        append_to_file(scraping_file, username)
+        logger.info(f"Added @{username} at {scraping_file}.txt")
+        return True, True
```

Comment:
should this not be False, False since we didn't interact with them?

Reply:
yes, you're right :)